llvm/include/llvm/IR/DataLayout.h

```diff
@@ -364,7 +373,7 @@ class DataLayout {
 
   /// Returns the maximum index size over all address spaces.
   unsigned getMaxIndexSizeInBits() const {
-    return getMaxIndexSize() * 8;
+    return getMaxIndexSize() * ByteWidth;
```

Comment:
Should there be something like a DL.bytesToBits() helper function? The inverse is a bit more tricky since we need to define how to handle non-multiples (round down, round up, assert).

Reply:
I never needed it, but it never occurred to me it might be useful.
I think we can add it later if we need it?